src/interpreter/mod.rs

```diff
@@ -20,6 +20,8 @@ use error::{EvalError, EvalResult};
 use memory::{Memory, Pointer};
 use primval::{self, PrimVal};
 
+mod iterator;
```

Comment:
This module name/filename is confusing given that it has no `Iterator` impl. `mod stepper` perhaps?